specification/customerlockbox/resource-manager/Microsoft.CustomerLockbox/preview/2018-02-28-preview/customerlockbox.json

```diff
@@ -0,0 +1,359 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-02-28-preview",
+    "title": "Customer Lockbox",
+    "description": "Azure Customer Lockbox API Reference"
+  },
+  "host": "management.azure.com",
+  "schemes": ["https"],
+  "consumes": ["application/json"],
+  "produces": ["application/json"],
+  "paths": {
```

Comment:
is there a need to cancel a request?

Reply:
At this time Lockbox does not support cancellation of requests. So I don't think our API will need to support that in this version.

Reply:
This is definitely an item that we are considering